src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -22,6 +23,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Target Name="_ComputeTargetFrameworkItems" Returns="@(InnerOutput)">
     <ItemGroup>
       <_TargetFramework Include="$(TargetFrameworks)" />
```

Comment:
Convention is generally that items have singular names--like `Compile`, `ProjectReference`, and so on. I agree that it's a bit confusing here . . . think it's worth changing?